StortSpelProjekt/Engine/src/Renderer/AnimatedModel.h

```diff
@@ -31,11 +31,15 @@ class AnimatedModel : public Model
     const ConstantBuffer* GetConstantBuffer() const;
     const std::vector<DirectX::XMFLOAT4X4>* GetUploadMatrices() const;
 
+    bool SetActiveAnimation(std::string animationName);
     void Update(double dt);
+    void toggleAnimation();
 
 private:
+    void initializeAnimation(SkeletonNode* node);   // Attach the currentStateTransforms of the animation to the skeleton.
     void updateSkeleton(float animationTime, SkeletonNode* node, DirectX::XMMATRIX parentTransform);
 
+    bool animationIsPaused = true;
```

Comment:
code standard

Reply:
Same. Would also prefer to have the initialisation in the constructor